pc/peer_connection.cc

```diff
@@ -3068,7 +3079,8 @@ bool PeerConnection::ReceiveRtp(uint8_t pt, bool enable_incoming) {
     if (enable_incoming) {
       rtp_transport->SetIncomingRtpEnabled(true);
     }
-    return rtp_transport->RegisterRtpDemuxerSink(demux_criteria, sink);
```

Comment:
Should we add another `// RingRTC change to receive RTP data` to this line?

Reply:
The whole function was added and already has the annotation.